src/screens/Gurbani/Navbar.tsx

```diff
@@ -0,0 +1,78 @@
+import { StackNavigationOptions } from '@react-navigation/stack'
+import React from 'react'
+import { Dimensions, Image, StyleSheet, View } from 'react-native'
```

Comment:
```suggestion
import React from 'react'
import { Dimensions, Image, StyleSheet, View } from 'react-native'
import { StackNavigationOptions } from '@react-navigation/stack'
```

Just because we have react import the first thing everywhere else :)

Reply:
Yep good catch!